src/playcanvas-anim.js

```diff
@@ -412,16 +412,25 @@ Object.assign(window, function () {
      * @param {pc.Vec3} vec3Scale
      * @param {object} output
      */
-    AnimationTarget.constructTargetNodes = function (root, vec3Scale, output) {
+    AnimationTarget.constructTargetNodes = function (root, vec3Scale, output, basePath) {
         if (!root)
             return;
 
+        if (!basePath) {
+            var currNode = root;
+            while(currNode.constructor !== pc.Entity) {
```

Comment:
is it worth testing for currNode === null? i'm imagining currNode reaching the summit of this tree without any node being an entity.

Reply:
👍